packages/expo-polyfills/esbuild.mjs

```diff
@@ -0,0 +1,37 @@
+/* global console */
+/* eslint no-console: "off" */
+import esbuild from 'esbuild';
+import { nodeExternalsPlugin } from 'esbuild-node-externals';
+import { writeFileSync } from 'node:fs';
+
+const options = {
+  entryPoints: ['./src/index.ts'],
+  bundle: true,
+  platform: '',
+  loader: { '.ts': 'ts' },
+  resolveExtensions: ['.ts'],
+  target: 'es2021',
+  tsconfig: 'tsconfig.json',
+  minify: true,
+  sourcemap: true,
+  plugins: [nodeExternalsPlugin()],
+  external: ['expo-crypto', 'expo-standard-web-crypto'],
```

Comment:
Shouldn't all of the `dependencies` be in this list?

```json
  "dependencies": {
    "base-64": "1.0.0",
    "expo-crypto": "12.4.1",
    "expo-standard-web-crypto": "1.6.0",
    "react-native-url-polyfill": "2.0.0",
    "text-encoding": "0.7.0"
  },
```

Reply:
Actually I don't think this is even needed, builds without this field on the config... using the `node-externals` plugin makes all imported packages external by default... cool.